src/main/java/com/mineboundteam/minebound/magic/OffensiveSpells/NecroticOffensiveSpell.java

```diff
@@ -0,0 +1,170 @@
+package com.mineboundteam.minebound.magic.OffensiveSpells;
+
+import java.util.List;
+
+import com.mineboundteam.minebound.MineBound;
+import com.mineboundteam.minebound.capabilities.ArmorNBTHelper;
+import com.mineboundteam.minebound.capabilities.PlayerSelectedSpellsProvider;
+import com.mineboundteam.minebound.capabilities.PlayerSelectedSpellsProvider.PrimarySpellProvider.PrimarySelected;
+import com.mineboundteam.minebound.capabilities.PlayerSelectedSpellsProvider.SecondarySpellProvider.SecondarySelected;
+import com.mineboundteam.minebound.config.IConfig;
+import com.mineboundteam.minebound.item.armor.ArmorTier;
+import com.mineboundteam.minebound.magic.ActiveSpellItem;
+import com.mineboundteam.minebound.magic.MagicType;
+import com.mineboundteam.minebound.magic.SpellType;
+
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.food.FoodData;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.level.Level;
+import net.minecraftforge.common.ForgeConfigSpec;
+import net.minecraftforge.common.ForgeConfigSpec.DoubleValue;
+import net.minecraftforge.common.ForgeConfigSpec.IntValue;
+import net.minecraftforge.event.entity.living.LivingDamageEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.config.ModConfigEvent;
+
+
+@Mod.EventBusSubscriber(modid = MineBound.MOD_ID)
+public class NecroticOffensiveSpell extends ActiveSpellItem {
+    public static DoubleValue FOOD_REDUCTION;
+
+    private final int manaCost;
+    private final double damageBoost;
+    private final double vampirePercent;
+
+    public final static String ACTIVE_TAG = MineBound.MOD_ID + ".necrotic_offensive.active";
+    public final static String TICK_TAG = MineBound.MOD_ID + ".necrotic_offensive.ticks";
+
+    public NecroticOffensiveSpell(Properties properties, NecroticOffensiveSpellConfig config) {
+        super(properties, config.LEVEL, MagicType.NECROTIC, SpellType.OFFENSIVE);
+        manaCost = config.MANA_COST.get();
+        damageBoost = config.DMG_BOOST.get();
+        vampirePercent = config.HEALTH_AMT.get();
+    }
+
+    @Override
+    public void use(ItemStack stack, Level level, Player player) {
+        stack.getOrCreateTag().putBoolean(ACTIVE_TAG, true);
+    }
+
+    @Override
+    public void onUsingTick(ItemStack stack, Level level, Player player) {
+        if(stack.getOrCreateTag().getBoolean(MineBound.MOD_ID + ".necrotic_offensive.active")) {
```

Comment:
Should use your `ACTIVE_TAG` variable here. Same in `releaseUsing`.

Reply:
Done